ninja/operation.py

```diff
@@ -31,6 +31,28 @@
 __all__ = ["Operation", "PathView", "ResponseObject"]
 
 
+def load_request(request: HttpRequest):
+    """
+    Loads a non-json PUT/PATCH request as a POST, reading data from request body
+    and placing them in request.POST and request.FILES.
+    This is required to fix the behaviour when we have a PUT/PATCH endpoint that also
+    receives a File as parameter, forcing the data to be sent as multipart/form-data and
+    making the ParamModel look up at request.POST and request.FILES
+    """
+    if request.method in ("PUT", "PATCH") and request.content_type != "application/json":
+        original_method = request.method
+        if hasattr(request, '_post'):
+            del request._post
+            del request._files
+        try:
+            request.method = "POST"
+            request.META['REQUEST_METHOD'] = 'POST'
+            request._load_post_and_files()
```

Comment:
I think this should just hit `request.FILES` rather than manually accessing an undocumented underscored method?
```suggestion
            request.FILES
```

Reply:
Correct me if i am wrong but this is probably due to how django handles `FILES`.

Quoting @deniswvieira, The alternative is to,

> Add some logic into _MultiPartBodyModel and FileModel to load the request body data like the Django does for POST request (essencially, replicate Django logic)

Which in turn adds churn

Reply:
@baseplate-admin `request.FILES` is a property of WSGIRequest in django that triggers _load_post_and_files if `._files` doesn't exist. Which it doesn't, because it was deleted a few lines earlier in this PR ;)
https://github.com/django/django/blob/main/django/core/handlers/wsgi.py#L104-L108